.cirrus.yml

```diff
@@ -153,7 +153,7 @@ task:
   ccache_cache:
     folder: '%CCACHE_DIR%'
   install_tools_script:
-    - choco install --yes --no-progress ccache --version=4.6.1
+    - choco install --yes --no-progress ccache --version=4.7.4
     - choco install --yes --no-progress python3 --version=3.9.6
```

Comment:
nit, realized after merge: I guess it should be fine to bump this as well, so that everything is still pinned but bumped to the latest version. (And add a comment that bumping to the latest in this file is fine?)

Finally, a two line patch could be squashed into one commit, but no big deal.